cmd/homi/setup/cmd.go

```diff
@@ -1064,22 +1081,32 @@ func WriteFile(content []byte, parentFolder string, fileName string) {
 	fmt.Println("Created : ", filePath)
 }
 
+func existGenTypeArg(genTypeArg string) bool {
+	for _, t := range Types {
+		if genTypeArg == t {
+			return true
+		}
+	}
+	return false
+}
+
 func findGenType(ctx *cli.Context) int {
+	// Klaytn-Node: genTypeFlag's default value is docker
+	genTypeFlag := ctx.String(genTypeFlag.Name)
+	if ctx.Args().Present() && existGenTypeArg(ctx.Args()[0]) {
+		genTypeFlag = ctx.Args()[0]
+	}
 	genType := TypeNotDefined
```

Comment:
Isn't this enough?
```suggestion
	genType := TypeNotDefined
	if ctx.Args().Present() {
		genTypeFlag = ctx.Args()[0]
	}
```

Reply:
I thought the `first argument` can be the unrelated option, not `genTpyeFlag`.
Is there anything I have missed?